### PR TITLE
pidhi

### PR DESCRIPTION
**Pull request by BryanEdison**

### CODE REVIEW

**Comment by slimtone** [comment]
definitely not done. still a lot to style but im tired so im stopping for the nite

**Comment by slimtone** [comment]
you can merge it if you want

File: components/LotSubmissionForm.js [hunk 1]
```diff
@@ -149,18 +152,26 @@ export default class LotSubmissionForm extends Component {
...
+						<Button rounded success onPress={() => { this.handleSubmit("brs") }}><Text>Request BayRide Supreme </Text></Button>
+
+
+
+=======
+					<View>
+						<Button style={style.button} rounded success onPress={() => { this.handleSubmit('brx') }}><Text style={{fontSize: 20}}>Request BayRide</Text></Button>
+						<Button style={style.button} rounded success onPress={() => { this.handleSubmit('brxl') }}><Text style={{fontSize: 20}}>Request BayRideXL</Text></Button>
+						<Button style={style.button} rounded success onPress={() => { this.handleSubmit('brs') }}><Text style={{fontSize: 20}}>Request BayRide Supreme </Text></Button>
+>>>>>>> 28b23fc0fba54e93bee14ddd9ead2ccebd581e81
```
Comment (RedPineapple) [inline components/LotSubmissionForm.js]: Unresolved Merge Conflict

File: components/ViewPhotos.js [hunk 2]
```diff
@@ -20,15 +22,13 @@ export default class ViewPhotos extends Component {
...
-					onPress={this._pickImage}
-				/>
+			// <FormLabel>Screenshot</FormLabel>
+			<View style={style.viewPhoto}>
+			<View><UploadPhotoButton label='Add a photo' color='rgb(0,175,115)' onPress={this._pickImage}/></View>
+
 				{image &&
 					<Image source={{ uri: image }} />}
-			</View>
+			// </View>
```
Comment (RedPineapple) [inline components/ViewPhotos.js]: Why is this View tag commented out? Where's the closing View tag for the one on line 26?

Comment (RedPineapple) [reply components/ViewPhotos.js]: It wasn't commented out, because of JSX comment rules, but why was it like that?

**Pull request merged**